package.json

```diff
@@ -3,15 +3,34 @@
   "version": "1.0.0",
   "description": "A simple inverted index application",
   "main": "inverted-index.js",
+  "engines": {
+    "node": "6.9.4",
+    "npm": "4.1.2"
+  },
   "dependencies": {
-    "gulp": "^3.9.1"
+    "browser-sync": "^2.18.6",
+    "coveralls": "^2.11.15",
+    "eslint": "^3.14.0",
+    "eslint-config-airbnb-base": "^11.0.1",
+    "eslint-plugin-import": "^2.2.0",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
+    "gulp-jasmine": "^2.4.2",
+    "gulp-jasmine-browser": "^1.7.1",
+    "gulp-livereload": "^3.8.1",
+    "jasmine-core": "^2.5.2",
+    "jasmine-node": "^1.14.5",
+    "request": "^2.79.0"
   },
   "devDependencies": {
-    "express": "^4.14.0",
-    "gulp": "^3.9.1"
+    "async": "^2.1.4",
+    "gulp": "^3.9.1",
```

Comment:
Nitpick: You have gulp listed in your dependencies so it should not be listed under devdependencies too.